### ISSUE TITLE
Type DirectoryPickerPage unavailable

### ISSUE THREAD

**Issue by ekari** [fuchsmich/notizenmd#8]
Hi,

running SailfishOS 3.4.0.24 on a Xperia XA2, I installed harbour-notizenmd-0.6.3-1.noarch.rpm from openrepos. Notizen.md starts ok, but from the pulldown menu, Settings do nothing. Output from the console:
```
sailfish-qml harbour-notizenmd
[D] unknown:0 - Using Wayland-EGL
[D] unknown:0 - Got library name:  "/usr/lib/qt5/qml/io/thp/pyotherside/libpyothersideplugin.so"
[D] read:14 - reading true
[D] onCompleted:18 - *.md,*.txt object 2
[W] unknown:9 - file:///usr/share/harbour-notizenmd/qml/pages/About.qml:9: ReferenceError: ta is not defined
[W] modifyPageStack:113 - Error while modifying page stack: Error: Error while loading page: file:///usr/share/harbour-notizenmd/qml/pages/Settings.qml:63 Type DirectoryPickerPage unavailable
file:///usr/share/harbour-notizenmd/qml/pages/DirectoryPickerPage.qml:7 module "org.nemomobile.contentaction" is not installed

[W] unknown:483 - file:///usr/lib/qt5/qml/Sailfish/Silica/PageStack.js:483: Error: Error while loading page: file:///usr/share/harbour-notizenmd/qml/pages/Settings.qml:63 Type DirectoryPickerPage unavailable
file:///usr/share/harbour-notizenmd/qml/pages/DirectoryPickerPage.qml:7 module "org.nemomobile.contentaction" is not installed
```
Am I missing something on the installation?

**Comment by fuchsmich** [comment]
Thanks for the report. Will check, what's wrong ASAP.

**Comment by fuchsmich** [comment]
Should be fixed with 0.6.4
Soon available on openrepos.

**Comment by ekari** [comment]
Yes, problem solved. Thanks!
